build.gradle

```diff
@@ -7,6 +7,8 @@ buildscript {
 
 plugins {
     id 'com.github.ben-manes.versions' version '0.28.0'
+    id 'com.github.johnrengelman.shadow' version '5.2.0'
```

Comment:
Needed to create a self-contained 'fat-jar' to tackle class-loading issue upon plugin loading.